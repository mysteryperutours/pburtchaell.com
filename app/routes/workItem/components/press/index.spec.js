import React from 'react';
import Press from './index';

const MOCK_PRESS_ITEMS = [{
  id: '0',
  title: 'Foo',
  url: 'http://foo.com'
}];

describe(Press.name, () => {
  it('renders component', () => {
    const component = (
      <Press
        items={MOCK_PRESS_ITEMS}
      />
    );

    expect(shallow(component)).toMatchSnapshot();
  });
});
