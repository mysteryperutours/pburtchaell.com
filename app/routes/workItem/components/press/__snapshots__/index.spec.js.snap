exports[`Press renders component 1`] = `
<section
  style={
    Object {
      "marginTop": "7rem",
    }
  }>
  <Row
    defaultColumn={false}
    elementType="div">
    <Column
      elementType="div"
      largeSize={3}
      scale="large"
      smallSize={4}
      style={Object {}}>
      <h5
        style={
          Object {
            "margin": "1rem",
            "textAlign": "right",
          }
        }>
        Press
      </h5>
    </Column>
    <Column
      elementType="div"
      largeSize={7}
      scale="large"
      smallSize={8}
      style={Object {}}>
      <div
        className="case-study-meta case-study-meta--press">
        <ul>
          <li>
            <div
              className="case-study-link">
              <a
                href="http://foo.com"
                title="Foo">
                \"Foo\"
              </a>
            </div>
            <div />
          </li>
        </ul>
      </div>
    </Column>
  </Row>
</section>
`;
